webpack.demo.js

```diff
@@ -0,0 +1,49 @@
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    'app': './demo/src/demo.ts',
+    'vendor': './demo/src/vendor.ts'
+  },
+  output: {
+    path: __dirname + '/demo/dist',
+    filename: '[name].bundle.js',
+    chunkFilename: '[name].bundle.js'
+  },
+  resolve: {
+    extensions: ['', '.ts', '.js']
+  },
+  devtool: 'eval',
+  module: {
+    loaders: [
+      {
+        test: /\.ts$/,
+        loader: 'ts-loader'
+      }
+    ],
+    noParse: [ /angular2\/bundles\/.+/ ]
+  },
+  plugins: [
+    new webpack.optimize.OccurrenceOrderPlugin(),
+    new webpack.optimize.DedupePlugin(),
+    new webpack.optimize.CommonsChunkPlugin({
```

Comment:
So as soon as you use commons chunk plugin all the benefits of excluding vendor go out of the window. My vote: let's not bother and keep webpack's config simple